src/platform/Linux/ConnectivityManagerImpl.cpp

```diff
@@ -938,7 +941,8 @@ CHIP_ERROR ConnectivityManagerImpl::_WiFiPAFPublish(ConnectivityManager::WiFiPAF
     wpa_supplicant_1_interface_call_nanpublish_sync(mWpaSupplicant.iface, args, &publish_id, nullptr, &err.GetReceiver());
 
     ChipLogProgress(DeviceLayer, "WiFi-PAF: publish_id: %u ! ", publish_id);
-    _GetWiFiPAF()->AddPafSession(publish_id);
+    WiFiPAFSession sessionInfo = { .id = publish_id };
+    ReturnErrorOnFailure(_GetWiFiPAF()->AddPafSession(PafInfoAccess::kAccSessionId, sessionInfo));
```

Comment:
Please do not use symbols with leading underscores. Singletons should look like `WiFiPAF()::GetInstance()` and they should return references not pointers.

Reply:
I will change it.